### ISSUE TITLE
``Init train iterator, skipping 0 elements` not progressing

### ISSUE THREAD

**Issue by KristofferTolboll2** [daniel-kukiela/nmt-chatbot#132]
i forked the project and inserted my own test/train data. I ran the `prepare_data.py` with no issues, but when i got the the `train.py`, it stucks at 
```
# Start step 0, lr 0.001, Tue Jun 11 12:18:16 2019
# Init train iterator, skipping 0 elements
```
after a while.. though the first part runs very smoothly. I'm on a 16GB RAM Lenovo, but with a default CPU, is that too little to run?

**Comment by ZotecXIX** [comment]
At this part in the process, the network is training. If you open up task manager, you should see your CPU being pinned (that is if you are training on your CPU). While training on your CPU you practically cant use your computer and it (the training) will be very slow. Highly recommend using a GPU system.

I'm no expert when it comes to this code, but this is most likely the problem.

**Comment by kaljitism** [comment]
Nope, it will take time to update the progress in steps done in terminal.

**Comment by ArjunR00T** [comment]
Check tensorboard and you can see ur neural network training

**Comment by iamdivyanshurai** [comment]
@acrpsm how ??

**Comment by iamdivyanshurai** [comment]
@KristofferTolboll2 after how much time did it continue to work after that line ??
please telll
